modules/lmdbbackend/lmdbbackend.cc

```diff
@@ -905,26 +906,37 @@ BOOST_SERIALIZATION_SPLIT_FREE(DomainInfo);
 BOOST_IS_BITWISE_SERIALIZABLE(ComboAddress);
 
 template <>
-std::string serializeToBuffer(const LMDBBackend::LMDBResourceRecord& lrr)
```

Comment:
i'm not sure this argument rename is actually an improvement, but I don't feel strongly about it

Reply:
I know. It's to solve a clang-tidy complaint about how the argument name is different in the original template vs. the specializations. But it's also not too bad `serialize<value>ToBuffer`.

Reply:
ah right